dove/tests/cmd_dove_ct_name.rs

```diff
@@ -0,0 +1,176 @@
+        execute(args, dove_folder.clone()).expect(&format!(
+            "[COMMAND] {}\r\n[FOLDER] {}",
+            &command_string,
+            dove_folder.to_str().unwrap()
+        ));
+        set_dependencies_local_move_stdlib(&project_folder);
+    }
+
+    // $ cargo run -- build
+    // $ dove build
+    {
+        let args = &["dove", "build"];
+        let command_string: String = args.join(" ").to_string();
+        execute(args, project_folder.clone()).expect(&format!(
+            "[COMMAND] {}\r\n[FOLDER] {}",
+            &command_string,
+            project_folder.to_str().unwrap()
+        ));
+        let args = &["dove", "build"];
+        let command_string: String = args.join(" ").to_string();
+        execute(args, project_folder.clone()).expect(&format!(
+            "[COMMAND] {}\r\n[FOLDER] {}",
+            &command_string,
+            project_folder.to_str().unwrap()
+        ));
+    }
+
+    // project_folder/scripts/sdemo.move
+    {
+        let mut path = project_folder.clone();
+        path.push("scripts");
+        path.push("sdemo.move");
+        write_all(
+            &path,
+            "script {
+                        fun main(_a:u64,_b:u64) { }
+                    }
+                    script {
+                        fun test_fun() { }
+                    }",
+        )
+        .unwrap();
+    }
+    // $ cargo run -- ct -n test_fun -f sdemo
+    // $ dove ct -n test_fun -f sdemo
+    {
+        let args = &["dove", "ct", "-f", "sdemo", "-n", "test_fun"];
+        let command_string: String = args.join(" ").to_string();
+        execute(args, project_folder.clone()).expect(&format!(
+            "[COMMAND] {}\r\n[FOLDER] {}",
+            &command_string,
+            project_folder.to_str().unwrap()
+        ));
+
+        let tx_path = {
+            let mut file = project_folder.clone();
+            file.push("target");
+            file.push("transactions");
+            file.push("test_fun.mvt");
+            file
+        };
+
+        assert!(
+            tx_path.exists(),
+            "Transaction not found: {}\r\n[Command] {}",
+            tx_path.to_str().unwrap(),
+            &command_string,
+        );
+
+        remove_file(&tx_path).unwrap();
+    }
+
+    // $ cargo run -- ct 'test_fun()' -f sdemo
+    // $ dove ct 'test_fun()' -f sdemo
+    {
+        let args = &["dove", "ct", "test_fun()", "-f", "sdemo"];
+        let command_string: String = args.join(" ").to_string();
+        execute(args, project_folder.clone()).expect(&format!(
+            "[COMMAND] {}\r\n[FOLDER] {}",
+            &command_string,
+            project_folder.to_str().unwrap()
+        ));
+
+        let tx_path = {
+            let mut file = project_folder.clone();
+            file.push("target");
+            file.push("transactions");
+            file.push("test_fun.mvt");
+            file
+        };
+
+        assert!(
+            tx_path.exists(),
```

Comment:
Shouldn't we check that another file is missing? Or will compiler build both transactions?